js/turtledefs.js

```diff
@@ -221,7 +221,7 @@ function createHelpContent() {
         'pitch': [_('The Pitch block specifies the pitch name and octave of a note that together determine the frequency of the note.'), 'documentation', 'note-value-block.svg'],
         'solfege': [_('Pitch can be specified in terms of do re mi fa sol la ti.'), 'documentation', 'note-value-block.svg'],
         'notename': [_('Pitch can be specified in terms of C D E F G A B.'), 'documentation', 'note-name-block.svg'],
-        'pitchnumber': [_('The Pitch Number block will play a pitch associated by its number, e.g, 1 for C, 7 for G.'), 'documentation', 'pitch-number-block.svg'],
+        'pitchnumber': [_('The Pitch Number block will play a pitch associated by its number, e.g, 0 for C, 7 for G.'), 'documentation', 'pitch-number-block.svg'],
```

Comment:
If it is 0 for C, it cannot be 7 for G.

Reply:
C = 0
C# = 1
D = 2
D# = 3
E = 4
F = 5
F# = 6
G = 7

BTW, one of my students made this project, which may be helpful.

https://musicblocks.sugarlabs.org/index.html?id=1544287859138573&run=True

Description: Student work. The student was tasked to create "Happy Birthday" entirely with pitch number. It would be analogous to playing "Happy Birthday" on the C string of an Ukulele. The numbers would be the frets. Next steps would be to break the melody up into separate actions.